### PR TITLE
[*] Technical - Fix linted deleted files issue

### PR DESCRIPTION
**Pull request by slimee**
This PR allows committing a file deletion.
Before that, `pre-commit-hook` linter was adding the deleted files in the `listModifiedFiles`

Here is an example of the issue. After deleting a file, the linter wanted to lint it.
Screenshot 2019-12-10 at 15 41 58

### CODE REVIEW

File: .eslint-bin/pre-commit-hook.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ function getFilesModified(callback) {
     }
 
     listFilesModified = status.files
+      .filter(change => change.index !== 'D')
```
Comment (VincentMolinie) [inline .eslint-bin/pre-commit-hook.js]: Actually it's even more than that, it would be nice to remove the file from the list file as it is useless now

Comment (slimee) [reply .eslint-bin/pre-commit-hook.js]: ? It is what it do, isn't it?
The added line do: filter files, do not take deleted ones

Comment (VincentMolinie) [reply .eslint-bin/pre-commit-hook.js]: Et les tests passent pas

File: CHANGELOG.md [hunk 2]
```diff
@@ -19,6 +19,7 @@
 - MySQL - Handle BIT(1) boolean columns and handle the buffer<01> value returned on join queries.
 - Technical - Fix unexpected test errors while running in the CI.
 - Technical - Fix database connection pool issue on tests.
+- Technical - Fix lint on deleted files issue.
```
Comment (VincentMolinie) [inline CHANGELOG.md]:
```suggestion
- Technical - Fix pre-commit hook when deleting a file.
```

**Pull request merged**